### ISSUE TITLE
Remove Always Pinned Notification

### ISSUE THREAD

**Issue by BeingDatta** [MCMrARM/revolution-irc#251]
First of all,

Thanks for such a beautiful, robust & standalone irc client. Just one request is it possible to disable always on pinned notification but keeping it running in background like other im apps.

Regards.

**Comment by MCMrARM** [comment]
Yes it is but only starting in Android 8. Go to Android notification settings and disable this notification.

**Comment by BeingDatta** [comment]
is it possible to make this available on nougat its annoying anyhow please communicate i mean if i build from source what i have to do to make it disable its very annoying destroying valuable screen real estate.

Regards

**Comment by BeingDatta** [comment]
is it possible to make this available on nougat its annoying anyhow please communicate i mean if i build from source what i have to do to make it disable its very annoying destroying valuable screen real estate.

Regards

**Comment by justjanne** [comment]
On Nougat this isn't possible. Android will simply stop and kill the app (so you lose connection and won't receive new messages) if you don't show such a notification.

To prevent this, you could alternatively use a bouncer such as ZNC or Quassel on a server, connect your mobile IRC client to it, and then your client wouldn't have issues with the connection being ended anymore.

**Comment by MCMrARM** [comment]
What could be done is lowering the notification priority on Nougat so that it is not shown as an icon in the status bar, I guess that should be added for Nougat and below.

**Comment by justjanne** [comment]
Yeah, that does work (we do that in Quasseldroid as well) :)
